tutorials/_index.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Tutorials"
+weight = 2
+sort_by = "weight"
+template = "sections/docs/chapters.html"
+aliases = ["docs/learn/"]
++++
+
+Tutorials and explanations for all the components of Urbit.
+
+## [Arvo](@/docs/learn/arvo/_index.md)
+
+Arvo is a clean-slate system software stack defined as a deterministic computer. An encrypted P2P network, %ames, runs on a functional operating system, Arvo, written in a strict, typed functional language, Hoon, which compiles itself to a combinator interpreter, Nock, whose spec gzips to 340 bytes.
+
+## [Hoon Tutorial](@/docs/learn/hoon-tutorial/_index.md)
+
+Hoon is a high-level, statically typed, functional programming language, custom-designed for use with Urbit. For a functional language it has a surprisingly imperative style, and mastering it doesn't require knowing any advanced mathematics. The Hoon Tutorial is the best place to start if you want to learn Urbit.
+
+## [Nock](@/docs/learn/nock/_index.md)
+
+Nock is the low-level functional programming language that is interpreted by Vere. You don't need to know Nock to write Hoon, or even to work on most parts of Vere.  But if you want to understand the foundations of Urbit and are mathematically-inclined, dive right in.
+
+## [Vere](@/docs/learn/vere/_index.md)
```

Comment:
Please amend all the relative links on these pages to their new locations — otherwise, as is will cause a fail on build.

Reply:
aye aye